DatadogRUM/Tests/RUMMonitor/Scopes/RUMViewScopeTests.swift

```diff
@@ -2537,8 +2538,7 @@ class RUMViewScopeTests: XCTestCase {
 
         // Then
         let rumViewWritten = try XCTUnwrap(featureScope.eventsWritten(ofType: RUMViewEvent.self).last, "It should send view event")
-        let baggageSent = try XCTUnwrap(featureScope.messagesSent().firstBaggage(withKey: RUMBaggageKeys.viewEvent))
-        let rumViewSent: RUMViewEvent = try baggageSent.decode()
-        DDAssertReflectionEqual(rumViewSent, rumViewWritten, "It must sent written event over message bus")
+        let rumViewInFatalErrorContext = try XCTUnwrap(fatalErrorContext.view)
+        DDAssertReflectionEqual(rumViewWritten, rumViewInFatalErrorContext, "It must update fatal error context with the view event written")
```

Comment:
💡 This change in `RUMViewScopeTests` is around asserting the update on `FatalErrorContext` instead of inspecting the baggage sent over message bus. The baggage is now tested in `FatalErrorContextTests` which streamlines things and better separates concerns.